File: Second Year/Semester 2/JAVA/Tasks/Section Tasks/Task 4/q1.java
import java.util.Scanner;

public class q1 {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int n;
        System.out.println("Enter number of elements");
        n = input.nextInt();

        int[] numbers = new int[n];
        
        int odd_counter = 0, even_counter = 0;

        take_array(numbers);

        for (int i = 0; i < n; i++) {
            even_counter = (numbers[i] % 2 == 0)? even_counter++:odd_counter++;
        }

        System.out.println("Even : " + even_counter);
        System.out.println("ODD : " + odd_counter);
        
    }

    static void take_array(int[] a){
        Scanner input = new Scanner(System.in);
        System.out.println("Enter array elements");
        for (int i = 0; i < a.length; i++) {
            int number = input.nextInt();
            a[i] = number;
        }
    }
}

File: First Year/Semister 2/C/Tasks/Task 01/Q1.c
#include <stdio.h>

int main()
{
    int n1, n2, n3, n4;

    printf("Enter the numbers:\n");
    scanf("%d %d %d %d", &n1, &n2, &n3, &n4);

    if (n1>n2 && n1>n3 && n1>n4)
    {
        printf("\n%d is the biggest", n1);
    }
    else if (n2>n1 && n2>n3 && n2>n4)
    {
        printf("\n%d is the biggest", n2);
    }
    else if (n3>n1 && n3>n2 && n3>n4)
    {
        printf("\n%d is the biggest", n3);
    }
    else if (n4>n2 && n4>n3 && n4>n1)
    {
        printf("\n%d is the biggest", n4);
    }

}